main.py

```
@@ -0,0 +1,37 @@
+from flask import Flask, request
+from utils import *
```

Ruff (0.11.9)

`from utils import *` used; unable to detect undefined names

(F403)